src/molecules/ContentBlock/ContentBlock.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import cn from 'classnames';
+import { Banner, Heading, Button } from '../../index';
+
+export interface Block {
+  type?: 'ContentBlock';
+  __type?: 'ContentBlock';
+  title: string;
+  text: string;
+  image: {
+    title: string;
+    url: string;
+  };
+  orientation: boolean;
+  buttonLink?: string;
+  buttonText?: string;
+}
+
+export interface ContentBlockProps {
+  block: Block;
```

Comment:
```suggestion
export interface ContentBlockProps extends Block {
```

Do we need to keep the nested structure, or could this be flatten down?